yarn-project/cli/src/cmds/l1/index.ts

```diff
@@ -34,7 +34,7 @@ export function injectCommands(program: Command, log: LogFn, debugLogger: Logger
     .description('Deploys all necessary Ethereum contracts for Aztec.')
     .addOption(l1RpcUrlsOption)
     .option('-pk, --private-key <string>', 'The private key to use for deployment', PRIVATE_KEY)
-    .option('--validators <string>', 'Comma separated list of validators')
+    .option('--validators <string>', 'Comma separated list of attestors')
```

Comment:
seems weird to have the flag be "validators" and have them described as "attestors"